spec/emoji_formatter.rb

```diff
@@ -1,17 +1,23 @@
 class EmojiFormatter < RSpec::Core::Formatters::ProgressFormatter
   # This registers the notifications this formatter supports, and tells
   # us that this was written against the RSpec 3.x formatter API.
-  RSpec::Core::Formatters.register self, :example_passed, :example_pending, :example_failed, :start_dump
+  RSpec::Core::Formatters.register(
+    self,
+    :example_passed,
+    :example_pending,
+    :example_failed,
+    :start_dump
+  )
 
   def example_passed(_notification)
-    output.print "💚 "
+    output.print '💚 '
   end
 
   def example_pending(_notification)
-    output.print "🙇 "
+    output.print '🙇 '
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.